"can we have two statements in a try block? i am new to programming, and am currently in product management. so i am learning the ropes. i tried this statement:..public class tuna { .    formatter f;.    public void createfile(){.    try{            .        f = new formatter(""help.text""); .        f.format(""%s%s"", ""firstname "",""lastname"");.    }..    catch (exception e){.        system.out.println(""you got an error "");.    }.}...now the first statement executes and a file is created, but the second statement does not execute by creating an entry into the file. ..at the same time when i created a method called createrecord() and inserted the f.format(..); statement it worked. ..can anyone tell me how all of this works?"
